action/config/validate.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package config
+
+import (
+	"fmt"
+	"os"
+)
+
+// Validate verifies the configuration provided.
+func (c *Config) Validate() error {
```

Comment:
This is where we add the new `Validate()` function.

The purpose of this function is to validate the configuration provided to the package.